### PR TITLE
ocamlPackages.parmap: 1.2 -> 1.2.3

### PR DESCRIPTION
**Pull request by superherointj**
ocamlPackages.parmap: 1.2 -> 1.2.3

### CODE REVIEW

**Review by bcdarwin — APPROVED** [review]
Looks good to me.

**Comment by bcdarwin** [comment]
@GrahamcOfBorg build ocamlPackages.parmap

File: pkgs/development/ocaml-modules/parmap/default.nix [hunk 1]
```diff
@@ -2,23 +2,25 @@
...
-  useDune2 = true;
+  version = "1.2.3";
 
   src = fetchurl {
     url = "https://github.com/rdicosmo/${pname}/releases/download/${version}/${pname}-${version}.tbz";
-    sha256 = "sha256-XUXptzD0eytaypaBQ+EBp4iVFRE6/Y0inS93t/YZrM8=";
+    sha256 = "sha256-W7T0vcbOsNirvrjY4GgFIZDTZds/g/hAZHIUcqkO6PE=";
   };
 
+  minimalOCamlVersion = "4.03";
```
Comment (vbgl) [inline pkgs/development/ocaml-modules/parmap/default.nix:12]: There is a typo here.

Comment (superherointj) [reply pkgs/development/ocaml-modules/parmap/default.nix]: Done.

**Comment by vbgl** [comment]
@GrahamcOfBorg build coccinelle

**Pull request merged**